Formula/luau.rb

```diff
@@ -0,0 +1,22 @@
+class Luau < Formula
+  desc "Fast, safe, gradually typed embeddable scripting language derived from Lua"
+  homepage "https://luau-lang.org"
+  url "https://github.com/Roblox/luau/archive/0.504.tar.gz"
+  sha256 "544b3107e1c53f8349b1e327efed55ac29cc20a2983dd23031ab07c5b6944633"
+  license "MIT"
+  head "https://github.com/Roblox/luau.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "cmake", "--build", ".", "--target", "Luau.Repl.CLI"
+    system "cmake", "--build", ".", "--target", "Luau.Analyze.CLI"
+    bin.install "luau", "luau-analyze"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DLUAU_BUILD_TESTS=OFF"
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```
Can you try this?

Reply:
It results in an empty installation error.

Reply:
Ok, we can replace the last line with
```ruby
bin.install "build/luau", "build/luau-analyze"
```